CRM/Report/Form.php

```diff
@@ -4521,10 +4521,13 @@ public function getPermissionedFTQuery(&$query, $alias = NULL, $return = FALSE)
     else {
       $contFTs = $liFTs = implode(',', array_keys($financialTypes));
     }
+    $temp = CRM_Utils_Array::value('civicrm_line_item', $query->_aliases);
     if ($alias) {
-      $temp = CRM_Utils_Array::value('civicrm_line_item', $query->_aliases);
       $query->_aliases['civicrm_line_item'] = $alias;
     }
+    elseif (!$temp) {
+      $query->_aliases['civicrm_line_item'] = 'civicrm_line_item_civireport';
```

Comment:
What is impact of specifying an alias for a table that is not used in a report? Are we confident that this will not cause changes to the queries for those reports?